files/class-vip-filesystem-stream-wrapper.php

```diff
@@ -105,6 +114,11 @@ public function __construct( API_Client $client = null, $protocol = null ) {
 		}
 
 		$this->protocol = $protocol ?: self::DEFAULT_PROTOCOL;
+
+		$this->debug_mode = false;
+		if ( defined( 'WP_DEBUG' ) ) {
```

Comment:
Since WP_DEBUG can be set to false:

```suggestion
		if ( defined( 'WP_DEBUG' ) && true === WP_DEBUG ) {
```